core/app/models/mno_enterprise/tenant.rb

```diff
@@ -1,6 +1,55 @@
 module MnoEnterprise
   class Tenant < BaseResource
+    # == Attributes ===========================================================
     property :created_at, type: :time
     property :updated_at, type: :time
+#   property :frontend_config
+
+    # jsonapi-client build the url with attributes[:primary_key]
```

Comment:
Can you add a line saying: "No primary key required here as this class is designed to hit the /tenant (singular resource) endpoint"